auth/src/main/java/org/jboss/pnc/bacon/auth/DirectKeycloakClientImpl.java

```diff
@@ -163,30 +165,23 @@ public Credential getCredentialServiceAccount(
         }
     }
 
-    private Credential refreshToken(Credential credential) {
-
-        try {
-            String keycloakEndpoint = keycloakEndpoint(credential.getKeycloakBaseUrl(), credential.getRealm());
-            HttpResponse<KeycloakResponse> postResponse = Unirest.post(keycloakEndpoint)
-                    .field("grant_type", "refresh_token")
-                    .field("client_id", credential.getClient())
-                    .field("refresh_token", credential.getRefreshToken())
-                    .asObject(KeycloakResponse.class);
-
-            KeycloakResponse response = postResponse.getBody();
-            Instant now = Instant.now();
-
-            return credential.toBuilder()
-                    .accessToken(response.getAccessToken())
-                    .accessTokenExpiresIn(now.plusSeconds(response.getExpiresIn()))
-                    .refreshToken(response.getRefreshToken())
-                    .refreshTokenExpiresIn(now.plusSeconds(response.getRefreshExpiresIn()))
-                    .build();
-
-        } catch (Exception e) {
-            log.error(e.getMessage());
-            return null;
```

Comment:
Remove this and throw `KeycloakClientException` instead, because `null` was never checked for, resulting in a possible NullPointerException`, but the `KeycloakClientException` is already caught.